web/src/main/scala/net/wiringbits/forms/ResendVerifyEmailFormData.scala

```diff
@@ -0,0 +1,40 @@
+package net.wiringbits.forms
+
+import net.wiringbits.api.models.SendEmailVerificationToken
+import net.wiringbits.common.models.{Captcha, Email}
+import net.wiringbits.webapp.utils.slinkyUtils.forms.{FormData, FormField}
+
+case class ResendVerifyEmailFormData(
+    email: FormField[Email],
+    captcha: Option[Captcha] = None
+) extends FormData[SendEmailVerificationToken.Request] {
+  override def fields: List[FormField[_]] = List(email)
+
+  override def formValidationErrors: List[String] = {
+    val emptyCaptcha = Option.when(captcha.isEmpty)("Complete the captcha")
+
+    List(
+      fieldsError,
+      emptyCaptcha
+    ).flatten
+  }
+
+  override def submitRequest: Option[SendEmailVerificationToken.Request] = {
+    val formData = this
+    for {
+      email <- formData.email.valueOpt
+      captcha <- formData.captcha
+    } yield SendEmailVerificationToken.Request(
+      email,
+      captcha
+    )
+  }
+}
+
+object ResendVerifyEmailFormData {
+  def initial(
```

Comment:
We need to add the "Complete the captcha" message to i18nMessages. 
A way to do this is creating a case class `Texts` inside the ResendVerifyEmailFormData object.
For example: 
```
object ResendVerifyEmailFormData {
  case class Texts(emptyCaptchaError: String)
  ...
}
```
Then we'll receive this case class Texts on our case class `ResendVerifyEmailData`, It'll look something like this:
```
case class ResendVerifyEmailFormData(
  texts: ResendVerifyEmailFormData,
  ...
)
```
This way we can replace the "Complete the captcha" message for texts.emptyCaptchaError

Reply:
Like this?

```
case class ResendVerifyEmailFormData(
    texts: Texts,
    email: FormField[Email],
    captcha: Option[Captcha] = None
) ...
```
```
object ResendVerifyEmailFormData {
  case class Texts(emptyCaptchaError: String)

  def initial(
      emptyCaptchaError: String,
      emailLabel: String
  ): ResendVerifyEmailFormData = ResendVerifyEmailFormData(
    texts = Texts(emptyCaptchaError),
    email = new FormField[Email](label = emailLabel, name = "email", required = true, `type` = "email")
  )
}
```

Reply:
That will work but I think it'd be easier to receive a instance of Texts as a parameter on the initial function instead of each text (one by one). This will allow us to add a new error text more easily 
Like this:
```
case class ResendVerifyEmailFormData(
    texts: ResendVerifyEmailFormData.Texts,
    email: FormField[Email],
    captcha: Option[Captcha] = None
) ...
```
```
object ResendVerifyEmailFormData {
  case class Texts(emptyCaptchaError: String)

  def initial(
      texts: ResendVerifyEmailFormData.Texts,
      emailLabel: String
  ): ResendVerifyEmailFormData = ResendVerifyEmailFormData(
    texts = texts,
    email = new FormField[Email](label = emailLabel, name = "email", required = true, `type` = "email")
  )
}
```